openslides_backend/action/actions/user/create_update_permissions_mixin.py

```diff
@@ -434,3 +462,84 @@ def _meetings_from_group_B_fields_from_instance(
         meetings: Set[int] = set(instance.get("is_present_in_meeting_ids", []))
         meetings.add(cast(int, instance.get("meeting_id")))
         return meetings
+
+
+class CreateUpdatePermissionsFailingFields(CreateUpdatePermissionsMixin):
+    def __init__(
+        self,
+        permstore: PermissionVarStore,
+        services: Services,
+        datastore: DatastoreService,
+        relation_manager: RelationManager,
+        logging: LoggingModule,
+        env: Env,
+        skip_archived_meeting_check: Optional[bool] = None,
+        use_meeting_ids_for_archived_meeting_check: Optional[bool] = None,
+    ) -> None:
+        self.permstore = permstore
+        self.user_id = self.permstore.user_id
```

Comment:
The `user_id` is set first thing in the `perform` method. Where do you need it earlier?